addons/registry/2.7.1/install.sh

```diff
@@ -211,3 +229,8 @@ function determine_registry_pvc_size() {
 
     export REGISTRY_PVC_SIZE=$registry_pvc_size
 }
+
+function registry_pvc_migrated() {
+    registry_pod=$( kubectl get pods -n kurl -l app=registry -o jsonpath='{.items[0].metadata.name}')
+    kubectl -n kurl logs $registry_pod -c migrate-s3  | grep "migration ran successfully" &>/dev/null
```

Comment:
`grep -q` rather than /dev/null will silence output

Reply:
can you just make sure that this function doesnt output a bunch of stuff to stderr say if the registry pod doesnt exist

Reply:
yeah i used try_1m instead of try1_stderr so we dont see errors when the registry pod restarts. But it echo's if the final attempt fails.